### DISCUSSION TITLE
Where and how Rust is used ?

### DISCUSSION THREAD

**Discussion by Aakash-ML** [xai-org/grok-1#234] [General]
The offical release page says it used rust and Jax . 
Not sure how and where /in which step rust was used ? Please provide details if anyone is aware of .

**Comment by anddea** [comment]
Grok is using Qdrant which is written in Rust.

https://github.com/qdrant/qdrant
https://x.com/qdrant_engine/status/1721097971830260030

Qdrant is actually used by other companies too:
https://x.com/simonw/status/1722011967886688696
